README.md

```diff
@@ -115,6 +115,10 @@ The `RewardManager` contract is responsible for creating the gnosis safes that a
 1. The `rewardee` or the owner of the prepaid card that executed the `registerRewardee` action.
```

Comment:
You can move it out. I put it there mainly because I wanted to check if our "assumed" order worked -- and it did. Maybe there is a use-case where the order of owners would change and we'd need to robustify how we transfer owners, but can be handled in separate pr.